test/functorch/test_vmap.py

```diff
@@ -3677,31 +3677,26 @@ def test_vmap_exhaustive(self, device, dtype, op):
         xfail('jiterator_2inputs_2outputs', device_type='cuda'),
         xfail('special.airy_ai'),
         xfail('clamp_min', ''),
-        xfail('special.bessel_j0'),
         xfail('sparse.sampled_addmm'),
-        xfail('special.bessel_y0'),
         xfail('special.chebyshev_polynomial_u'),
         xfail('special.modified_bessel_k1'),
         xfail('segment_reduce', 'offsets'),
-        xfail('special.bessel_j1'),
+        xfail('logspace', ''),
+        xfail('empty', ''),
```

Comment:
This `xfail` was removed in a PR recently (you will also have to remove it otherwise it will lead to test failure)